app/views/candidate_mailer/conditions_met.text.erb

```diff
@@ -1,9 +1,19 @@
 Dear <%= @application_form.first_name %>
 <% if @application_form.show_new_reference_flow? %>
 
-  <%= @course.provider.name %> has confirmed that you’ve met your conditions and they’ve checked your references.
+  <%= @course.provider.name %> has confirmed that you’ve met the conditions of your offer.
 
-  If they have not already, <%= @course.provider.name %> should contact you with details about enrolling on the course. Contact them directly if you have any questions.
+  They’ll let you know if they need further information before you can start training. Contact them if you have any questions.
+
+  You can sign into your account if you need to check the progress of your reference requests.
+
+  [Sign into your account](<%= candidate_magic_link(@application_form.candidate) %>).
+
+  #If you’re unable to start training in <%= @start_date %>
```

Comment:
@johndoates do we know if start date is a Month and Year or a Day Month and Year? If it's the latter, does this need to be 'on' start date?

Reply:
It's the same variable as the one used in app/views/candidate_mailer/unconditional_offer_accepted.text.erb. So it should show just month and year, as in this screenshot.

![CleanShot 2022-10-10 at 12 33 16@2x](https://user-images.githubusercontent.com/29047487/194857113-ef68026b-1f27-4aae-a505-6045a0f54b47.png)